perfkitbenchmarker/linux_benchmarks/iperf_benchmark.py

```diff
@@ -76,6 +76,9 @@ def Prepare(benchmark_spec):
       vm.AllowPort(IPERF_PORT)
     vm.RemoteCommand('nohup iperf --server --port %s &> /dev/null &' %
```

Comment:
In case it's useful: elsewhere we've gotten the PID directly via [`echo $!`](https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/traces/dstat.py#L83)